install.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/sh
+
+if [[ -n $1 ]] && [[ $1 == '--scripts' ]]; then
+  echo '💡  Installing scripts…'
+
+  PATH_LINE='export PATH=$PATH:$HOME/.git-semantic-commits:$PATH'
+  BASHRC=~/.bashrc
+  ZSHRC=~/.zshrc
+
+  if [ -f $BASHRC ]; then
+    if ! grep -Fxq "$PATH_LINE" $BASHRC
+    then
+      echo
+      echo "📝  Adding path to $BASHRC"
+      echo $PATH_LINE >> $BASHRC
+      source $BASHRC
+    fi
+  fi
+
+  if [ -f $ZSHRC ]; then
+    if ! grep -Fxq "$PATH_LINE" $ZSHRC
+    then
+      echo
+      echo "📝  Adding path to $ZSHRC"
+      echo $PATH_LINE >> $ZSHRC
+      source $ZSHRC
+    fi
+  fi
```

Comment:
Line 10 to line 28 is quite a bit of duplication. Can this be extracted in a function of some sort?